src/module.zig

```diff
@@ -176,28 +187,112 @@ pub fn Objects(comptime T: type) type {
             try dead.resize(allocator, data.capacity, true);
             try generation.ensureUnusedCapacity(allocator, 1);
 
+            if (options.track_fields) {
+                try _updated.ensureUnusedCapacity(allocator, 1);
+                _updated.appendAssumeCapacity(try std.bit_set.DynamicBitSetUnmanaged.initEmpty(allocator, @typeInfo(T).@"struct".fields.len));
+            }
+
             const index = data.len;
             data.appendAssumeCapacity(value);
             dead.unset(index);
             generation.appendAssumeCapacity(0);
+
             return @bitCast(PackedID{
                 .type_id = objs.internal.type_id,
                 .generation = 0,
                 .index = @intCast(index),
             });
         }
 
-        pub fn set(objs: *@This(), id: ObjectID, value: T) void {
+        /// This function will set all fields of an object.
+        ///
+        /// Note that this function has no tracking, and will not trigger
+        /// any internal mach functions.
+        ///
+        /// Example: core.windows.setAllRaw(id, value) will NOT trigger the Platform
+        /// to update the window's size, etc.
```

Comment:
I'd like to avoid the mach.Core-specific example in the docs here. Also, I don't like that we say `will not trigger any internal mach functions` because that makes the module system sound way more magical than it is.

Instead, we can do something like:

```
/// Sets all fields of the given object to the given value.
///
/// Unlike setAll(), this method does not respect any mach.Objects tracking
/// options, so changes made to an object through this method will not be tracked.
```